runtime-config-linux.md

```diff
@@ -169,11 +169,11 @@ Optionally, cgroups limits can be specified via `resources`.
         },
         "blockIO": {
             "blkioWeight": 0,
-            "blkioWeightDevice": "",
-            "blkioThrottleReadBpsDevice": "",
-            "blkioThrottleWriteBpsDevice": "",
-            "blkioThrottleReadIopsDevice": "",
-            "blkioThrottleWriteIopsDevice": ""
+            "blkioWeightDevice": null,
+            "blkioThrottleReadBpsDevice": null,
+            "blkioThrottleWriteBpsDevice": null,
+            "blkioThrottleReadIopsDevice": null,
+            "blkioThrottleWriteIopsDevice": null
```

Comment:
There doesn't seem to be much point in having these examples if they're all `null`.  Maybe drop some of them completely, and then set the remaining ones to something more useful to show the new syntax?